src/xhr.js

```diff
@@ -65,10 +65,17 @@ var XHR = {
                 onreadystatechange = undefined;
 
                 // TODO(cory): have the notifier log an internal error on non-200 response codes
+                var jsonResponse = RollbarJSON.parse(request.responseText);
                 if (request.status === 200) {
-                  callback(null, RollbarJSON.parse(request.responseText));
+                  callback(null, jsonResponse);
                 } else if (Util.isType(request.status, 'number') &&
-                            request.status >= 400 && request.status < 600) {
+                  request.status >= 400 && request.status < 600) {
+
+                  if (request.status == 403 && console) {
+                    // likely caused by using a server access token, display console message to let
+                    // user know
+                    console.log(jsonResponse.message);
```

Comment:
To be consistent with the other log messages, this should start with `"[Rollbar]: "` e.g. https://github.com/rollbar/rollbar.js/blob/master/src/notifier.js#L33